docs/release.md

```diff
@@ -0,0 +1,81 @@
+# Release Process
+
+This document outlines the release process of the load-test tool. It is codified in [a Playbook](https://community.mattermost.com/playbooks/playbooks/95pa6a8fcp8p8xsn4frqejcide) that the team follows monthly in the Community server. If you don't have access to the playbook, please ping the team.
+
+## Requirements
+
+This release process follows [the release process of the self-hosted version of Mattermost](https://handbook.mattermost.com/operations/research-and-development/product/release-process/feature-release).
+
+The requirements for this release process are the following:
+- The monthly performance comparisons, where we compare the Mattermost version about to be released with the latest stable one, are based on the RC-1 version of the new release.
+- We need a stable tested version of the load-test tool to run the performance comparisons.
+- The RC-1 version of Mattermost is cut at around T-14[^1].
```

Comment:
I assume this is implicitly understood by everyone here but just want to highlight that there's always the case RC-1 won't be enough if we found a performance regression that either needs fixing or reverting.

Reply:
> Do you mean that the finally released version may be RC-2 or higher?

Yeah, and it may imply having to do a second comparison as well.

Reply:
I was gonna clarify this in the document, but on second thought, that would be part of the performance comparison process, not part of the release process. Once the load-test version is released, we will use that one to run the performance comparison as many times as needed on RC-1and following. If we ever need to revert a feature from the load-test tool after RC-1, we can plan that ad-hoc.